addons/web/static/src/views/form/form_compiler.js

```diff
@@ -175,6 +175,13 @@ export class FormCompiler extends ViewCompiler {
         return hasContent ? buttonBox : null;
     }
 
+    compileButton(el, params) {
+        const compiled = super.compileButton(el, params);
+        compiled.setAttribute("disableViewButtons", "props.disableViewButtons");
+        compiled.setAttribute("enableViewButtons", "props.enableViewButtons");
+        return compiled;
```

Comment:
not sure about this. For now, this is done imperatively (see view_button_hook). It would probably be a good idea to keep the same mechanism (this one or the other, whatever, but the same in both cases)

Reply:
KISS